x/bank/internal/types/genesis.go

```diff
@@ -1,18 +1,87 @@
 package types
 
+import (
+	"bytes"
+	"encoding/json"
+	"sort"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/exported"
+)
+
 // GenesisState is the bank state that must be provided at genesis.
 type GenesisState struct {
-	SendEnabled bool `json:"send_enabled" yaml:"send_enabled"`
+	SendEnabled     bool             `json:"send_enabled" yaml:"send_enabled"`
+	GenesisBalances []GenesisBalance `json:"genesis_balances" yaml:"genesis_balances"`
+}
+
+// Genesis Balance is a struct that pairs an sdk.AccAddress with an accounts balance at genesis
+type GenesisBalance struct {
+	Address sdk.AccAddress
```

Comment:
Does this need struct tags? i.e. `yaml:"address"` etc..

Reply:
Technically, no, but I'll add them.